Fakebook.Posts/Fakebook.Posts.UnitTests/Model.Tests/Comment.Tests.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using Fakebook.Posts.DataAccess.Mappers;
+using Xunit;
+
+namespace Fakebook.Posts.Models.Comment.UnitTests
+{
+    public class Comment
+    {
+
+    }
+}
```

Comment:
what's this file for?
(fyi, it can get annoying later on if you reuse an identifier for a type and for a namespace (Comment in this case), sometimes C# will get confused between them and require you to type extra things)

Reply:
Removed